src/pfe/portal/routes/projects/remoteBind.route.js

```diff
@@ -396,7 +396,7 @@ async function recursivelyListFilesOrDirectories(getDirectories, absolutePath, r
  * @return 404 if the project was not found
  * @return 500 if there was an error
  */
-router.post('/api/v1/projects/:id/bind/end', async function (req, res) {await bindEnd(req, res)});
+router.post('/api/v1/projects/:id/bind/end', bindEnd);
```

Comment:
You've updated `openapi.yml` for bind end, did you mean to enable validateReq here as well? (And also for `/upload`.)
(If you didn't please ignore this comment.)